dev/index.ts

```diff
@@ -90,7 +90,8 @@ setTimeout(() => {
 setTimeout(() => {
   const id = contacts[between({ min: 0, max: contacts.length - 1 })].id;
   console.log('[Modified Contact]:', id);
-  contactsService.modifyContact(id, {
+  contactsService.modifyContact({
```

Comment:
Did you test your new mnemonics functions? I don't see them here in the dev/index.ts file.

Reply:
I only checked that they were added to the contact correct, not on the functions on their own.